docs/docs/.vuepress/config.js

```diff
@@ -18,195 +19,8 @@ module.exports = {
-            internal: true,
-            badge: "New",
-            badgeVariation: "primary",
-          },
-          {
-            text: "v1",
-            internal: true,
-            link: "https://pipedream.com/docs/v1",
-          },
-        ],
-      },
-    ],
-    smoothScroll: false,
-    // if your docs are not at the root of the repo:
-    docsDir: "docs/docs",
-    editLinks: true,
-    // custom text for edit link. Defaults to "Edit this page"
-    editLinkText: "Help us improve this page! Submit an edit on Github",
-    sidebar: [
-      "/",
-      "/quickstart/",
-      {
-        title: "Workflows",
-        children: 
-        [
-          "/workflows/",
-          "/workflows/steps/",
-          "/workflows/steps/triggers/",
-          "/workflows/events/",
-          "/components/actions/",
-          "/workflows/concurrency-and-throttling/",
-          "/environment-variables/",
-          "/workflows/settings/",
-          "/workflows/networking/",
-          "/migrate-from-v1/",
-        ]
-      },
-      "/sources/",
-      "/connected-accounts/",
-      "/user-settings/",
-      {
-        title: "Reference: Code, APIs + CLI",
-        children: [
-          {
-            title: "Writing Code in Workflows",
-            type: "group",
-            initialOpenGroupIndex: 1,
-            children: [
-              "/code/",
-              {
-                title: "Node.js",
-                type: "group",
-                children: [
-                  "/code/nodejs/",
-                  "/code/nodejs/auth/",
-                  "/code/nodejs/http-requests/",
-                  "/code/nodejs/working-with-files/",
-                  "/code/nodejs/async/",
-                ],
-              },
-              "/code/python/",
-              "/code/go/",
-              "/code/bash/",
-              "/destinations/",
-              "/scheduling-future-tasks/",
-            ],
-          },
-          {
-            title: "Authoring Components",
-            type: "group",
-            children: [
-              "/components/",
-              "/components/quickstart/nodejs/actions/",
-              "/components/quickstart/nodejs/sources/",
-              "/components/api/",
-              "/components/guidelines/",
-              "/pipedream-axios/",
-            ],
-          },
-          {
-            title: "CLI",
-            type: "group",
-            children: ["/cli/install/", "/cli/login/", "/cli/reference/"],
-          },
-          {
-            title: "APIs",
-            type: "group",
-            children: [
-              "/api/overview/",
-              "/api/auth/",
-              "/api/rest/",
-              "/api/rest/webhooks/",
-              "/api/rest/rss/",
-              "/api/rest/workflow-errors/",
-              "/api/sse/",
-            ],
-          },
-          {
-            title: "Integrations",
-            type: "group",
-            children: [
-              "/apps/all-apps/",
```

Comment:
I remember you asking me about these, but they appear to have been removed in the new nav.

Reply:
Can we add this section back in?